### DISCUSSION TITLE
Training Log 2022-08-07

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#91] [Training Logs 训练日志]
# CPM-Live Training Log (August, 7)

Time: August, 7 2022 16:00

Recorder: @zh-zheng

## Loss
- Begin: 2.27
- End: 2.25
	
## Completed Data
- $\approx$ 949.37GB

## Average Grad Norm
- 0.47

## Progress
- 101.94%

## Comment

Since we drastically shrank the batch size, the fluctuation of training loss became significantly larger🧐. It's not a big deal!
